src/app/app.module.ts

```diff
@@ -1,18 +1,41 @@
 import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule }   from '@angular/forms';
 import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
 
 import { AppComponent } from './app.component';
-import { CoreComponent } from './core/core.component';
+import { VehiclesManagementComponent } from './feature/vehicles-management/vehicles-management.component';
+import { VehiclesService } from './core/vehicles/vehicles.service';
+import { VehiclesRestService } from './core/vehicles/vehicles-rest.service';
+import { HttpClientModule } from '../../node_modules/@angular/common/http';
+import { NewVehicleFormComponent } from './shared/forms/new-vehicle-form/new-vehicle-form.component';
+import { NewVehicleComponent } from './feature/new-vehicle/new-vehicle.component';
+import { TableModule } from '../../node_modules/primeng/table';
+import {AccordionModule} from 'primeng/accordion';     //accordion and accordion tab
```

Comment:
Dzięki za komenta, nie sposób się domyśleć XD

Reply:
Usunięto.